css files/loginAndRegistration.css

```diff
@@ -136,3 +136,29 @@ a:hover {
     border-radius: 10px;
     -webkit-box-shadow: inset 0 0 6px rgba(0, 0, 0, .3);
 }
+
+/* ===PASSWORD METER=== */
+meter {
+    height: 10px;
+    width: 98%;
+    margin: 0;
+}
+
+meter::-webkit-meter-bar {
+    background: rgba(0, 0, 0, 0.1) none;
+}
+
+meter[value="1"]::-webkit-meter-optimum-value { background: red; }
+meter[value="2"]::-webkit-meter-optimum-value { background: yellow; }
+meter[value="3"]::-webkit-meter-optimum-value { background: orange; }
+meter[value="4"]::-webkit-meter-optimum-value { background: green; }
```

Comment:
change the green to the hex code  (Use the dark accent and post a screenshot)
Main green color: 05D197
Dark accent: 2DB58F
Darker accent: 02845F